rasa_core/training/dsl.py

```diff
@@ -46,6 +46,9 @@ def _parse_item(self, line):
             example.data["true_intent"] = intent
             return example
 
+        raise ValueError("Encountered invalid end-to-end format for message "
```

Comment:
please add a link to the documentation where to find help on how the format looks like